exatomic/gaussian/tests/test_output.py

```diff
@@ -87,6 +119,12 @@ def setUp(self):
         self.uo2 = Output(resource('g09-uo2.out'))
         self.mam3 = Output(resource('g09-ch3nh2-631g.out'))
         self.mam4 = Output(resource('g09-ch3nh2-augccpvdz.out'))
+        # need two because of the current limitations in the parse_frequency code
```

Comment:
This should not be an issue with the current changes in the parse_frequency. Need to change

Reply:
Leaving it in place as it gives another example that should work as the other is a special case.